src/AIconLabel.cpp

```diff
@@ -10,16 +10,15 @@ AIconLabel::AIconLabel(const Json::Value &config, const std::string &name, const
     : ALabel(config, name, id, format, interval, ellipsize, enable_click, enable_scroll) {
   event_box_.remove();
   box_.set_orientation(Gtk::Orientation::ORIENTATION_HORIZONTAL);
-  box_.set_spacing(8);
-  box_.add(image_);
+  if (iconEnabled()) {
+    box_.set_spacing(8);
+    box_.add(image_);
+  }
   box_.add(label_);
   event_box_.add(box_);
 }
 
-auto AIconLabel::update() -> void {
-  image_.set_visible(image_.get_visible() && iconEnabled());
```

Comment:
Why removing this line?

Reply:
Hidden picture has size, and i added
```
if (iconEnabled()) {
    box_.set_spacing(8);
    box_.add(image_);
  },
```
and I added it so as not to add a picture at all
![Screenshot_20221202_0816](https://user-images.githubusercontent.com/33456625/205220343-a97eb5d7-943e-4195-a821-f76f9e769d13.png)